products/dbt-core.md

```diff
@@ -19,9 +19,16 @@ auto:
 # eol(x) = releaseDate(x) + 1 year
 # See https://docs.getdbt.com/docs/dbt-versions/core
 releases:
+-   releaseCycle: "1.7"
+    releaseDate: 2023-11-02
+    support: true
+    eol: 2024-11-02
+    latest: "1.7.0"
+    latestReleaseDate: 2023-11-02
+
 -   releaseCycle: "1.6"
     releaseDate: 2023-07-31
-    support: true
+    support: 2023-11-02
```

Comment:
The table at https://docs.getdbt.com/docs/dbt-versions/core#latest-releases says 1.6 is still under active support.

Reply:
Looks like they forget to update the table, because as per their release policy: 

> Critical Support: When a newer minor version is available, we will transition the previous minor version into "Critical Support." Subsequent patches to that older minor version will be "security" releases only, limited to critical fixes related to security and installation.

Reply:
Filed a PR upstream. https://github.com/dbt-labs/docs.getdbt.com/pull/4399

Merging this.